src/wordpress-playground/__tests__/migration-tests.ts

```diff
@@ -0,0 +1,312 @@
+import * as phpLoaderModule from '../../../build/php-7.4.node.js';
+import { startPHP } from '../../php-wasm/php-node';
+import { existsSync, rmSync, readFileSync } from 'fs';
```

Comment:
TypeScript is unhappy about `rmSync` – let's use `rmdirSync` instead

```suggestion
import { existsSync, rmdirSync, readFileSync } from 'fs';
```